external/harvestPackages/harvestPackages.props

```diff
@@ -1,48 +1 @@
<PackageReference Include="System.Diagnostics.DiagnosticSource">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.IO.FileSystem.AccessControl">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.IO.Packaging">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.IO.Pipes.AccessControl">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.IO.Ports">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Json">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Memory">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Net.Http.WinHttpHandler">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Numerics.Vectors">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Reflection.DispatchProxy">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Reflection.Metadata">
  <Version>1.5.0</Version>
</PackageReference>
<PackageReference Include="System.Reflection.TypeExtensions">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Runtime.CompilerServices.Unsafe">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Security.AccessControl">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Security.Cryptography.Cng">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Security.Cryptography.OpenSsl">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Security.Cryptography.Pkcs">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Security.Cryptography.ProtectedData">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Security.Cryptography.Xml">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Security.Permissions">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Security.Principal.Windows">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.ServiceProcess.ServiceController">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Text.Encoding.CodePages">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Text.Encodings.Web">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Threading.AccessControl">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.Threading.Tasks.Dataflow">
  <Version>4.8.0</Version>
</PackageReference>
<PackageReference Include="System.Threading.Tasks.Extensions">
  <Version>4.5.0</Version>
</PackageReference>
<PackageReference Include="System.ValueTuple">
  <Version>4.5.0</Version>
</PackageReference>
</ItemGroup>
</Project>

```

Comment:
Can you fix the line endings?

Reply:
Yes it is really difficult to tell what changed in this file.

Reply:
I see, the encoding was messed up with this file, which is what causes all of the issues since I just opened and saved it as UTF-8 as the rest of them which caused the changes. I'll revert that back and saved the file with the previous encoding.